d2l-input-textarea.html

```diff
@@ -67,6 +67,7 @@
 				font-family: inherit;
 				line-height: inherit;
 				text-align: inherit;
+				overflow: hidden;
```

Comment:
Does this still allow a scrollbar to be displayed when the max height of the container is exceeded and a scrollbar is required?

Reply:
Shoot, no. You can scroll by moving the cursor up and down beyond the visible content, but there is no visible scroll bar. Back to the drawing board...

Reply:
Seems a little odd that the scroll bar would display in this state.  What's causing the dimensions to be exceeded (as compared to other browsers)?

Reply:
Oh - IE defaults the overflow differently.

Reply:
Yup, it's not the dimensions.

Reply:
Is this OK to merge now? @dbatiste

Reply:
@mdulat I imagine it's ok - the only thing i would check is what is the default overflow on the mirror div? If that is also defaulting to scroll, then you might need to apply this style to the mirror too otherwise it can lead to resizing inconsistencies.